React/Views/RCTSliderManager.m

```diff
@@ -30,6 +30,13 @@ - (UIView *)view
 
 static void RCTSendSliderEvent(RCTSlider *sender, BOOL continuous)
 {
+  if (sender.step > 0 &&
+    sender.step <= (sender.maximumValue - sender.minimumValue)) {
+    int step = ((int)((sender.value + sender.step / 2) / sender.step) * sender.step);
+    [sender setValue: step
```

Comment:
I'm a little surprised that this works. Doesn't it prevent you from dragging the slider, since it would keep snapping back to the previous step?

If it does work, that's cool, but I'm a bit concerned that it's relying on an undocumented feature, and may break in a future iOS release.

Would it possibly be safer to send the stepped value in the event callback, but only actually update the slider value when continuous = false (i.e. when the user has stopped dragging).